2017/07/01_playing_with_time.py

```diff
@@ -72,6 +74,9 @@ def date_to_string(date_or_datetime: Union[date, datetime]) -> str:
     :param date_or_datetime: a date or datetime object to convert
     """
 
+    date_string = date_or_datetime.isoformat()
+
+    return date_string[:10]
```

Comment:
Interesting solution here. I would have reached for `strptime()`, but this works too. Note that this *wouldn't* work if we wanted to do non-iso formatted months and dates, without the leading zero for numbers less than ten